packages/api/lambdas/sf-semaphore-down.js

```diff
@@ -12,14 +10,25 @@ const {
   hasQueueAndExecutionLimit
 } = require('../lib/message');
 
+/**
+ * Determine if Cloudwatch event is a Step Function state update.
+ *
+ * @param {Object} event - A Cloudwatch event object
+ * @returns {boolean} - True if event is a Step Function state update.
+ */
+const isSfExecutionEvent = (event) => event.source === 'aws.states';
+
 /**
  * Determine if workflow is in a terminal state.
  *
- * @param {Object} message - A workflow message object
+ * @param {Object} status - A Step Function execution status
  * @returns {boolean} - True if workflow is in terminal state.
  */
-const isTerminalMessage = (message) =>
-  message.meta.status === 'failed' || message.meta.status === 'completed';
+const isTerminalMessage = (status) =>
+  status === 'FAILED'
```

Comment:
There are a lot of cases where we do a check like this. I think we may want to add an `isOneOf()` function to `@cumulus/common/utils`

Reply:
Would that be better than just doing `['FAILED', 'COMPLETED', 'TIMED_OUT', 'ABORTED'].includes(status)`?

Reply:
I think it makes it more readable.